pkg/registry/rbac/helpers.go

```diff
@@ -20,9 +20,16 @@ import (
 	"k8s.io/apimachinery/pkg/api/meta"
 	"k8s.io/apimachinery/pkg/conversion"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/registry/rest"
```

Comment:
For other reviewers, this package already imports other k8s.io/apisever/ packages so I'm assuming adding this import isn't an issue.